contents/docs/migrate/export-events.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Export events
+sidebar: Docs
+showTitle: true
+---
+
+At PostHog we believe in your right to own your data - this means being able to export the raw events whenever you want.
+
+There are several ways to export your events depending on your use case:
+
+1. Use the PostHog UI to filter the events table and then click "Export". Great for quickly getting access to the events you need. Limited to 10,000 events.
+2. Use the events API. Great for one-off smaller exports. Limited to a 1 day date range and 10,000 events.
+3. Use a data export app. Great for a very large number of events and you can stream new data to keep your destination up to date. No limits.
```

Comment:
The repetition of "Great for" makes me wonder if this should be a table?

|method|use when||
|-|-|-|
|PostHog UI|blah|3500|
|PostHog API|a different blah|1day, 3500|
|Apps|another different blah|No limits|

Reply:
great point, I'll switch this to a table next week